runtime/plugin/health.vim

```diff
@@ -1 +1,7 @@
-command! -nargs=* CheckHealth call health#check([<f-args>])
+function! s:complete(lead, _line, _pos) abort
+  return filter(map(glob($VIMRUNTIME.'/autoload/health/*', 1, 1),
```

Comment:
@mhinz Best way to do this is using `globpath()` (do not forget to escape commas). Second-best is concatenating with `fnameescape($VIMRUNTIME)` (does not work on Windows in some cases; AFAIR in some Neovim issue there was an agreement that such Vim issues should be fixed even if fixing them will break compatibility, but I do not know the state). This variant is inappropriate because it does not work if `$VIMRUNTIME` contains any glob character including things like `$word`.